vendors/pc/boards/windows/aws_demos/config_files/aws_demo_config.h

```diff
@@ -65,4 +65,16 @@
 /* Send AWS IoT MQTT traffic encrypted to destination port 443. */
 #define democonfigMQTT_AGENT_CONNECT_FLAGS             ( mqttagentREQUIRE_TLS | mqttagentUSE_AWS_IOT_ALPN_443 )
 
+#define democonfigMEMORY_ANALYSIS
+
+#ifdef democonfigMEMORY_ANALYSIS
+    #define demoMEMORY_ANALYSIS_STACK_DEPTH_TYPE    UBaseType_t
+    #define demoMEMORY_ANALYSIS_MIN_EVER_HEAP_SIZE( x )     xPortGetMinimumEverFreeHeapSize()
+    #if ( INCLUDE_uxTaskGetStackHighWaterMark == 1 )
+        #define demoMEMORY_ANALYSIS_STACK_WATERMARK( x )    uxTaskGetStackHighWaterMark( x )
+    #else
+        #define demoMEMORY_ANALYSIS_STACK_WATERMARK( x )    ( x )
```

Comment:
Instead of defining this macro (x), I would set it to a constant number or NULL. 

The reason is that, in this branch we don't seem to have a way to get lowest ever stack. Returning user input without checking/casting may result confusion.

Reply:
Okay that makes sense